source/applications/advanced/create_depth_map.py

```diff
@@ -12,57 +13,90 @@
 from utils.paths import get_sample_data_path
 
 
-def _main():
-
-    app = zivid.Application()
+def _point_cloud_to_cv_z(point_cloud):
+    """Get depth map from frame.
 
-    filename_zdf = Path() / get_sample_data_path() / "Zivid3D.zdf"
-    print(f"Reading {filename_zdf} point cloud")
-    frame = zivid.Frame(filename_zdf)
+    Args:
+        frame: Zivid point cloud
 
-    # Getting the point cloud
-    depth_map = frame.point_cloud().copy_data("z")
+    Returns:
+        OpenCV Depth map
```

Comment:
I don't know what an "OpenCV Depth map" is, but what you are returning here is just a "numpy.ndarray". All OpenCV images in Python are just Numpy arrays.

Reply:
Do you think I should just leave 'Depth map'? Or should I specify that it is a numpy.ndarray as well?

Reply:
The point was to emphesize that it is opencv compatible. But I get your point.

Reply:
I have changed to _Depth map (HxWx1 darray)_
I left 'OpenCV' in comments/descriptions of the samples.